openpgp.d.ts

```diff
@@ -337,13 +338,20 @@ interface Config {
   rejectMessageHashAlgorithms: Set<enums.hash>;
   rejectPublicKeyAlgorithms: Set<enums.publicKey>;
   rejectCurves: Set<enums.curve>;
+  hardwareKeys?: HardwareKeys;
 }
 export var config: Config;
 
 // PartialConfig has the same properties as Config, but declared as optional.
 // This interface is relevant for top-level functions, which accept a subset of configuration options
 interface PartialConfig extends Partial<Config> {}
 
+declare abstract class HardwareKeys {
```

Comment:
I think this should be an interface, as it's more generic. We are using abstract classes if we are extending them as part of the definitions or if we need other OOP features (e.g. `instanceof HardwareKeys`), which doesn't seem to be the case?